api-docs/rst/dev-guide/api-operations/methods/post-create-policy-v1.0-tenantid-groups-groupid-policies.rst

```diff
@@ -207,21 +207,21 @@ This table shows the body parameters for the request:
 |                    |             |exclusive with the ``cron`` parameter.     |
 +--------------------+-------------+-------------------------------------------+
 |[*].\               |Number       |The percent change to make in the number   |
-|**changePercent**   |*(Optional)* |of servers in the scaling group. If this   |
-|                    |             |number is positive, the number of servers  |
+|**changePercent**   |*(Optional)* |of units in the scaling group. If this     |
+|                    |             |number is positive, the number of units    |
 |                    |             |will increase by the given percentage. If  |
```

Comment:
number of units increases by the given percentage    (Avoid use of future tense per Rackspace style guidelines.)